cla_public/templates/base.html

```diff
@@ -180,6 +182,9 @@
         ga(values["name"] + '.linker.set', 'anonymizeIp', true);
         ga(values["name"] + '.linker:autoLink', ['www.gov.uk']);
       }
+      if(localeLanguage === 'cy') {
+        ga(values["name"] + '.set', 'language', 'en_cy')
+      }
```

Comment:
Is `en_cy` correct?  I'd have thought either `cy_GB` or simply `cy`.
Also, this is setting it to Welsh if it is already Welsh?  Does that achieve anything?  I would have thought we'd need to account for the Korean situation, by setting that to English.  
Maybe adding
` else { ga(values["name"] + '.set', 'language', 'en') }`

Reply:
Not sure tbh...I used en_cy to match what was asked in requirements. I would be more than happy to change that to cy, I think that would make it easier to understand.

To answer your 2nd question - I had to do it like this because Google Analytics doesn't care about the locale set, it only looks at the language in the browser. So I have to tell manually tell Google Analytics to set the language field as Welsh depending on the locale.

And yes, I have added that in my last commit. This will fix the Korean situation.